WordPress/src/main/java/org/wordpress/android/util/WPLegacyMigrationUtils.java

```diff
@@ -144,10 +145,10 @@ public static boolean hasSelfHostedSiteToMigrate(Context context) {
         }
     }
 
-    private static List<SiteModel> getSelfHostedSitesFromDeprecatedDB(Context context) {
+    static List<SiteModel> getSelfHostedSitesFromDeprecatedDB(Context context) {
```

Comment:
It looks like we [do update it](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/1939af38af3d07f620a0ab507cfee0c584a63bff/fluxc/src/main/java/org/wordpress/android/fluxc/network/xmlrpc/site/SiteXMLRPCClient.java#L166) when newly fetching self-hosted sites in FluxC (through `FETCH_SITES_XML_RPC`, which WPAndroid calls in `SignInFragment`).

But we don't call `FETCH_SITES_XML_RPC` for migrated sites, so we don't update the flag.

I think it would make the most sense to init `SiteModel.mIsVisible` as `true`, and only worry about changing it for REST sites.